main.py

```diff
@@ -0,0 +1,32 @@
+from app.animal import Animal, Carnivore, Herbivore
+
+
+if __name__ == "__main__":
+    lion = Carnivore("Simba")
+    print(len(Animal.alive) == 1)
```

Comment:
Code Efficiency: You can use the `assert` statement to check if a certain condition is met. It is a good practice to use assertions in your code, especially for tests. For example, you can rewrite this line as `assert len(Animal.alive) == 1`.